### ISSUE TITLE
Reverting Water Extractor: with semi-reasonable points

### ISSUE THREAD

**Issue by C4nc3ER** [Anuken/Mindustry-Suggestions#2128]
**Describe what you would like changed, and why.**
As simple as it is, reverting the water extractor... nerf? Balancing adjustment? Whatever you'd consider it, the metaglass addition, while not a catastrophic change, was certainly an unexpected one.

**Describe the changes you want to propose. Include possible alternatives.**
As simple as it is, removing the metaglass, or:

Adding a smaller, slower pumping water extractor that requires less power. 3 water per second at most, as laser drills can benefit majorly from max water.

*"Now, why can't you just get metaglass ezpz and make water extractor anyway?"*

In PvP, the water extractor can play a big role for late-game, getting Thorium faster than your enemy can get you T4/T5 sooner. The metaglass is an added hurdle.

*"But doesn't T3 need metaglass too-"*

**Also** consider that steam generators, a good early game power source, in schematics, typically used water extractors. With metaglass, it's more mid-game, when there are better alternatives, such as the Thermal Generator.

*"Thermal Generators don't work without heat, duh. Some maps don't have it. And about that T3-"*

Most maps do have those heat blocks, especially in PvP maps. Metaglass is a hurdle that's easily avoided with a smaller water extractor.

*"But why a smaller water extractor? Wouldn't that punt itself in the stratosphere because the regular one is much better? And please address T3-"*

Well, it's because it's similar to the progression of drills. The Laser Drill isn't overshadowed by the Airblast Drill because it locks necessary progression. With the smaller water extractor, it can be the key to power with Steam Generators, which can power Kilns, etc.

***"WHAT ABOUT TIER 3, YOU DUNCE"***

...yeah kinda overlooked that one. Sure you'd need metaglass for tier 3 units, but most PvP games go that far, at least in Ranked. That's changing, I believe, but I'm not an expert on it.

tl;dr: should probably read it but ok, metaglass is an addition that effected a lot of PvP progression and progression in game, in general. there are plenty of alternatives and this is an argument against it
